release-schema.json

```diff
@@ -50,10 +50,38 @@
           "maximum": 1,
           "minimum": 0
         },
+        "competitiveMaximumPercentage": {
+          "title": "Maximum percentage of total value for competitive subcontracting",
+          "description": "Maximum percentage of the contract value the procuring entity asks the contractor to subcontract to third parties through a competitive procedure.",
```

Comment:
Is it "asks" or "requires"? Also, start with "The " and add "that" before "the procuring entity", to make it easier to parse the sentence into clauses.

Reply:
[Article 21, 4) ](https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32009L0081&from=EN#d1e2623-76-1)

> Member States may provide that the contracting authority/entity may ask or be required to **ask** the successful tenderer to subcontract to third parties a share of the contract.

But also:

> The successful tenderer shall award subcontracts corresponding to the percentage which the contracting authority/entity requires it to subcontract in accordance with the provisions of Title III.

I guess it boils down to a requirement. I wish EU lawmakers stuck to may/could/should/must, as in W3C recommendations!

Reply:
Done